thorlcr/slave/thslavemain.cpp

```diff
@@ -285,19 +283,58 @@ bool UnregisterSelf(IException *e)
     return false;
 }
 
+void signalInFuture(int signo, unsigned timeout)
```

Comment:
I know it's seconds, but it's not clear from the prototype/parameter names, perhaps rename param to timeoutSecs, or method to signalInSecs

Reply:
don't know where else this method might be useful, but probably belongs in somewhere like jexcept

Reply:
I will change timeout to timeoutsec.
I can move this into jlib if you want.

Reply:
changed to timeoutSec

Reply:
I can move signalInFuture() into jexcept.cpp if you think its better there.

Reply:
not a big deal, but yes think it better to group the code there.

Reply:
ok, moved into jexcept.cpp